register_to_database.php

```diff
@@ -79,12 +79,12 @@
 			$mail->isSMTP();
 			$mail->Host = 'smtp.gmail.com';
 			$mail->SMTPAuth = true;
-			$mail->Username = 'schrutedwight600@gmail.com'; // Gmail address which you want to use as SMTP server
-			$mail->Password = 'Kingkong@1234'; // Gmail address Password
```

Comment:
Please ignore this.. This was an earlier commit, then I didn't realized it.. Now it's been replaced with empty string.